README.md

```diff
@@ -14,10 +14,9 @@ A curated list of Minio community projects inspired by [awesome-go](https://gith
 * [minio-dist-boshrelease](https://github.com/shinji62/minio-dist-boshrelease) - Cloud Foundry [BOSH](https://bosh.io) is an open source tool chain for release engineering, deployment and lifecycle management of large scale distributed services. minio-dist-boshrelease will help easily install the [distributed version of Minio](https://docs.minio.io/docs/distributed-minio-quickstart-guide).
 * [minio-dokku](https://github.com/slypix/minio-dokku) - Dockerfile to run Minio on [Dokku](http://dokku.viewdocs.io/dokku/) PaaS.
 * [sloppy.io](https://sloppy.io/) - sloppy.io is the fastest way to deploy your docker container online. We provide the infrastructure and workflow to run container applications and micro services. Access our platform over the web, via CLI or our own API. Check out [Deploying Minio to sloppy.io](https://sloppy.io/deploying-minio-to-sloppy-io/) for further info! 
-
+* [Deis Workflow](https://deis.com/workflow/) - an open source Platform-as-a-Service for [Kubernetes](https://kubernetes.io/). Workflow uses Minio by default to store all internal application metadata and database backups.
```

Comment:
Why this is change is Deis not IaaS anymore? @Shashwatsh ?

Reply:
??

Reply:
with IaaS, you can create, manage, delete VMs/Containers While PaaS simplifies workload deployment, it also restricts a business's flexibility to create the environment that they want. 
Deis is a PaaS.

Reply:
also deis is deprecated

Reply:
If deis is deprecated we should remove it from the docs here @Shashwatsh

Reply:
but it's just got deprecated, like this month and they'll give maintenance update for the next 6 months and so many people are still using it so i think it should be here for the next few months

Reply:
Oh okay understood makes sense.

Reply:
There is #77 now, mentioning Hephy Workflow which continues the work